text/1216-bang-type.md

```diff
@@ -191,7 +191,7 @@ So why do this? AFAICS there are 3 main reasons
     `Result<T, E>` to satisfy a trait but we know that the method will always
     succeed.
 
-    For example, here's a saner implementation of `FromStr` for `String` than
+    For example, here's a saner implementation of `FromStr` for `String` that
     currently exists in `libstd`.
```

Comment:
I think this one is correct as-is, as the RFC is written from the perspective of the feature not existing.  (And the current version still uses `ParseError` because back-compat.)

Reply:
Yes, I agree. The proposed change here is semantic and not just a typo fix.

Reply:
Oops, indeed. I'm sorry, I should have read that more attentively, before considering it as a mistake. I changed it back to the original.